en/docs/concepts/user-management/user-management.md

```diff
@@ -0,0 +1,25 @@
+# Introduction to User Management 
+
+In a typical organization, there are several employees who are responsible for accomplishing a versatile set of tasks on a daily basis with the help of different applications. Most of the times, even the same application is used in different ways by employees who have differenet job roles. Managing who accesses which application for what purposes can be an overhead for the administration. This is why user management is important. 
```

Comment:
times--> time

differenet --> different

Reply:
I think we can add a little bit more about why user management is important and make it a separate sub topic. Let's start this page with a simple definition of what we mean by user management? 

after explaining the concept, let's dive into how user management works with WSO2 IS specifically. 

(check out some competitor docs for ideas on structure : https://www.okta.com/blog/2019/01/user-management/)